src/registrar/admin.py

```diff
@@ -294,6 +294,26 @@ def contact(self, obj: models.Contact):
 
     contact.admin_order_field = "first_name"  # type: ignore
 
+    # Read only that we'll leverage for CISA Analysts
+    analyst_readonly_fields = [
+        "user",
+    ]
+
+    def get_readonly_fields(self, request, obj=None):
+        """Set the read-only state on form elements.
+        We have 1 conditions that determine which fields are read-only:
+        admin user permissions.
+        """
+
+        readonly_fields = list(self.readonly_fields)
```

Comment:
(Q) why are we doing a list conversion here? Is this of type dict?

Reply:
So I mirrored this from other sections, but I think you're right -- it's a type of dict and we want things in list form for the admin page!